bigflow/dataflow.py

```diff
@@ -12,31 +13,35 @@
 
 class BeamJob(Job):
     def __init__(
```

Comment:
Don't change the order of the arguments because you break the API backward compatibility.